documentation/book/con-reconciliation.adoc

```diff
@@ -0,0 +1,7 @@
+[id='reconciliation-{context}']
+= Reconciliation
+
+Although the operator reacts to all notifications about the desired cluster resources received from the {ProductPlatformName}
+cluster, if the operator is not running, or if a notification is not received for any reason, the desired resources will get out of sync with the state of the running {ProductPlatformName} cluster.
+
+In order to handle failovers properly, a periodic reconciliation process is executed by the Cluster Operator so that it can compare the state of the desired resources with the current cluster deployments in order to have a consistent state across all of them.
```

Comment:
This needs to link to the Configuration variable `STRIMZI_FULL_RECONCILIATION_INTERVAL_MS` which configures how often does the reconciliation happen.